src/main/java/com/facebook/openwifirrm/ucentral/UCentralClient.java

```diff
@@ -137,7 +140,10 @@ public static void verifySsl(boolean enable) {
 	 * The access token obtained from uCentralSec, needed only when using public
 	 * endpoints.
 	 */
-	private String accessToken;
+	private WebTokenResult accessToken;
+
+	/** Time window to refresh token in seconds. */
+	private int TOKEN_REFRESH_WINDOW_S = 21600;
```

Comment:
It's only documented in the task description [T127668163](https://www.internalfb.com/intern/tasks/?t=127668163)

Reply:
I guess what I want to know is if owsec agress with that view or if it's something that's left up to the microservices. I would think the latter doesn't make sense - so that 6 hour seems a bit arbitrary

Reply:
Yup it's arbitrary. This should be moved into `RRMConfig`

Reply:
seconded, if it's arbitrary it should be configurable